ClosedXML.Tests/Extensions/FontBaseExtensionsTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using ClosedXML.Excel;
+using ClosedXML.Extensions;
+using NUnit.Framework;
+using SkiaSharp;
+using System.Collections.Generic;
+
+namespace ClosedXML.Tests.Extensions
+{
+    public class FontBaseExtensionsTests
+    {
+        [Test]
+        [Platform("Win", Reason = "Expectation only fits windows system because the font calibri isn't available on other OSs")]
+        public void GetHeightCalibri()
```

Comment:
Would prefer the test case name "Calibri" separated by an underscore from the SUT method in the test case method name.
I have not seen all of the test-code of the project but I have seen that the underscore is used in several places as well as with no underscore.

Reply:
I am on the !prevent risks side" of "Benefit of reduced effort preventing upgrades" vs  "risk to break anything"... 
packages used in production code get updated using dependabot
dev time dependecies are blacklisted independabot to prevent noise and can be upgraded by any dev

Reply:
I like the "Should naming standard" https://methodpoet.com/unit-test-method-naming-convention/#Should_naming_standard... other tests seem to follow the "test method is the same like SUT method"... thats no thing I stick with